### PR TITLE
Don't fail cluster cleanup when prefix not found

### PR DESCRIPTION
**Pull request by bertiethorpe**
- Exit gracefully when cluster prefix not found for deletion
- Echo clusters for debug
- Change schedule to 9pm

### CODE REVIEW

File: .github/workflows/nightly-cleanup.yml [hunk 1]
```diff
@@ -52,13 +52,23 @@ jobs:
...
+          else
+            echo "Found clusters: $CI_CLUSTERS"
+            echo "ci_clusters=${CI_CLUSTERS}" >> GITHUB_ENV
+          fi
         shell: bash
       
       - name: Delete clusters if control node not tagged with keep
         run: |
           . venv/bin/activate
+          if [[ -z "${CI_CLUSTERS}" ]]; then
```
Comment (wtripp180901) [inline .github/workflows/nightly-cleanup.yml:67]: this was set to `ci_clusters` in previous step, I think `CI_CLUSTERS` will be empty in this step

File: .github/workflows/nightly-cleanup.yml [hunk 2]
```diff
@@ -52,13 +52,23 @@ jobs:
...
+            echo "No clusters to delete."
+            exit 0
+          fi
+          
           for cluster_prefix in ${CI_CLUSTERS}
```
Comment (wtripp180901) [inline .github/workflows/nightly-cleanup.yml:72]: see above

**Review by wtripp180901 — CHANGES_REQUESTED** [review]
Bug in the last step

**Comment by bertiethorpe** [comment]
Tested successfully:

https://github.com/stackhpc/ansible-slurm-appliance/actions/runs/11893406516/job/33138326985

**Review by wtripp180901 — APPROVED** [review]
LGTM

**Pull request merged**